File: 0001_0100/0013/0013.py
with open("input.txt", "r") as f, open("output.txt", "w") as q:
    s, t = f.read().split()
    a = sum(1 for i in range(4) if s[i] in t and t.index(s[i]) == i)
    b = sum(1 for i in range(4) if s[i] in t) - a
    q.write(str(a) + ' ' + str(b))

File: 0001_0100/0013/0013.go
package main

import (
	"fmt"
	"os"
	"strings"
)

func main() {
	f, _ := os.Open("input.txt")
	q, _ := os.Create("output.txt")
	var b, k byte
	var s, t string
	fmt.Fscan(f, &s, &t)
	for i, c := range s {
		if c == rune(t[i]) {
			b++
		} else if strings.ContainsRune(t, c) {
			k++
		}
	}
	fmt.Fprint(q, b, k)
}
